lib/cluster/index.ts

```diff
@@ -0,0 +1,663 @@
+import ClusterAllFailedError from '../errors/ClusterAllFailedError'
+import {defaults, noop} from '../utils/lodash'
+import ConnectionPool from './ConnectionPool'
+import {NodeKey, IRedisOptions, normalizeNodeOptions} from './util'
+
+var Deque = require('denque')
```

Comment:
Some of those requires can probably be moved to import (namely node and some of your own).
Everything should probably be `const`